docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+
+## 6. Optionally, Install the Yaapu Telemetry script
+
+You may wish to install the [Yaapu Telemetry script](https://github.com/yaapu/FrskyTelemetryScript) which uses the CRSF telemetry messages which mLRS generates on your Tx module from the MAVLink data it receives when the Rx Serial Link Mode is set to "MAVLink" or "MAVLinkX".
+
+- If you won't always use a ground station to set MAVLink stream rates when it starts up, be sure to set your ArduPilot [Stream Rate](CRSF.md#stream-rates) parameters.
+
+- See the [Yaapu Wiki](https://github.com/yaapu/FrskyTelemetryScript/wiki) and [mLRS CRSF Telemetry](CRSF.md) page for details.
+
+## 7. Connect a ground station to your mLRS Tx module
+
+There are MAVLink ground station applications available for all common platforms including Windows, MacOS, Linux, Android, and iOS. We won't cover selecting a platform or installing a ground station here.  But we will provide some hints and suggestions for popular connection methods.
+
+Wireless links are possible for all three of the recommended Tx modules.
+
+Bluetooth is very handy for mobile devices and is recommended for the MatekSys Tx modules since they include an HC04 module which is automatically configured by mLRS.  Bluetooth is also supported by the [M5 Stamp Pico Mate](https://shop.m5stack.com/products/m5stamp-pico-diy-kit) which is recommended for use with the [FrSky R9M Tx module](FRSKY_R9.md#esp32-wireless-bridge).
+- The [HC04](MATEKSYS.md#tx-module-hc-04-bluetooth-notes) will be discoverable soon after being powered on.
+- First, pair your HC04 (pin: 1234) or esp32 with your device or computer.  Look for a device which includes "mLRS" in its name.
+- iOS devices do not support the Bluetooth Serial Port Profile and QGroundControl does not support Bluetooth LE so QGroundControl will not connect over Bluetooth on iOS.  It has been reported that the SidePilot App works on iOS.
+- QGroundControl works well with Bluetooth on Android, Linux or Windows.  Create a new Bluetooth type connection via Application Settings -> Comm Links -> Add.
+- Mission Planner is recommended for Windows.  Connect to the com port associated with the Bluetooth connection after pairing.
+
+UDP or TCP over WiFi is another popular wireless option.
+- All three of the recommended Tx modules could be connected to an external esp32 board such as the M5 Stamp boards running the [Wireless Bridge](WIRELESS_BRIDGE.md) sketch.
+- All popular ground stations can connect via UDP
+- The standard UDP port for MAVLink is 14550.
+
+All three of the recommended Tx modules include serial ports which may be used to connect to your ground station.
+- Since PC serial ports are full RS232 ports which use higher voltage levels than the logic level serial connections on Tx modules, a USB ttl serial adapter is used to connect a Tx serial port to a PC or mobile device.
```

Comment:
I don't think there is anyone out there who would even know where to find a RS232 port on their computer and if would know how to connect the mLRS serial wires to an ugly sub-d and if would know how to get any porgram on the PC running using it ... (except few old nerds, but I would struggle with the last step too)

pl just drop the RS232 reference completely. 
Maybe "Use a USB-TTL adapter to connect the Tx module's serial port to ..."

Reply:
Some PCs do still have RS232 ports (I'm typing on one) which they may label as "serial" or "RS232 serial" and outside of our little hobby world, RS232 serial USB adapters might be more common than TTL level.  (I also have a RS232 USB adapter plugged into my home automation server).  I think it's important to note the voltage difference somewhere so someone new to the hobby doesn't end up damaging their hardware.  I'll change the wording a bit though.  BTW, I freely admit I am an old nerd.

Reply:
I find that refernce to RS232 most odd, but really your call :)